src/Core/System/Snippet/SnippetPatterns.php

```diff
@@ -0,0 +1,31 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\System\Snippet;
+
+use Shopware\Core\Framework\Log\Package;
+
+/**
+ * @internal
+ */
+#[Package('discovery')]
+abstract class SnippetPatterns
+{
+    /**
+     * Locale pattern based on BCP 47,
+     * restricted to ISO 639-1 (2-letter) language codes.
+     * Excludes 3-letter prefixes like `ger` or `eng`.
+     */
+    public const LOCALE_PATTERN_BCP47_ISO639_1 =
```

Comment:
Should we rename this constant? In my opinion, variable names should not dictate the content.